sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/Uri.java

```diff
@@ -54,12 +58,14 @@ public String getURIAsString() {
         return this.uriAsString;
     }
 
-    /***
-     * Attention: This is only used for fault injection to easier decide whether the address is primary address.
-     * @param primary
-     */
     public void setPrimary(boolean primary) {
```

Comment:
This is only called right after a Uri is created and removed the fault injection comment because using it now for the diagnostics